src/main/java/xyz/nucleoid/bedwars/game/active/shop/BwItemShop.java

```diff
@@ -36,40 +40,41 @@ public static ShopUi create(ServerPlayerEntity player, BwActive game) {
     }
 
     private static void addNavbar(ShopBuilder shop, ServerPlayerEntity player, BwActive game, int pageIndex) {
-        addNavigationEntry(shop, player, game, Items.END_STONE, new LiteralText("Blocks"), pageIndex == 1, BwItemShop::createBlocks);
-        addNavigationEntry(shop, player, game, Items.IRON_SWORD, new LiteralText("Melee Weapons"), pageIndex == 2, BwItemShop::createMelee);
-        addNavigationEntry(shop, player, game, Items.IRON_CHESTPLATE, new LiteralText("Armor"), pageIndex == 3, BwItemShop::createArmor);
-        addNavigationEntry(shop, player, game, Items.IRON_PICKAXE, new LiteralText("Tools"), pageIndex == 4, BwItemShop::createTools);
-        addNavigationEntry(shop, player, game, Items.BOW, new LiteralText("Archery"), pageIndex == 5, BwItemShop::createArchery);
-        addNavigationEntry(shop, player, game, Items.POTION, new LiteralText("Utilities and Potions"), pageIndex == 6, BwItemShop::createUtils);
+        addNavigationEntry(shop, player, game, Items.END_STONE, "blocks", pageIndex == 1, BwItemShop::createBlocks);
+        addNavigationEntry(shop, player, game, Items.IRON_SWORD, "melee", pageIndex == 2, BwItemShop::createMelee);
+        addNavigationEntry(shop, player, game, Items.BOW, "archery", pageIndex == 3, BwItemShop::createArchery);
+        addNavigationEntry(shop, player, game, Items.IRON_PICKAXE, "tools", pageIndex == 4, BwItemShop::createTools);
+        addNavigationEntry(shop, player, game, Items.IRON_CHESTPLATE, "armor", pageIndex == 5, BwItemShop::createArmor);
+        addNavigationEntry(shop, player, game, Items.POTION, "utils", pageIndex == 6, BwItemShop::createUtils);
```

Comment:
Can I ask why this has been reordered?

Reply:
Thought it'd look better, but I have to admit this was much more of a pet-peeve of mine rather than a general issue. Reverting that